README.md

```diff
@@ -98,12 +89,11 @@ For command line help, visit our wiki page on Neat’s [command line interface](
   npm install --save bourbon-neat
   ```
 
-1. If you’re using [Eyeglass](http://eyeglass.rocks), skip to Step 3. Otherwise, you’ll need to add Bourbon and Neat to your node-sass `includePaths` option. `require("bourbon-neat").includePaths` is an array of directories that you should pass to node-sass. How you do this depends on how node-sass is integrated into your project. You will also need to ensure that Bourbon is in the `includePaths` passed to node-sass.
```

Comment:
Was it intended to remove “`require("bourbon-neat").includePaths` is an array of directories that you should pass to node-sass.”?